cfpb/capital-framework
# Update the docs site for v5.0.0 release

v5.0.0 released an update to cf-icons for SVG icons. This docs update
includes those changes as well as all the docs for all the components
that depend on cf-icons.

## Additions

- All the SVG icons

## Changes

- Updated the docs for cf-icons and all the components that depend on it.

## Testing

- Ensure you're not symlinking the components
- run `npm install`
- run `gulp build`
- run `npm start`
Enjoy the new icons in the docs
